asreview/webapp/src/ProjectComponents/SetupComponents/DataComponents/AddDataset.js

```diff
@@ -105,10 +105,10 @@ const AddDataset = (props) => {
 
   React.useEffect(() => {
     if (props.mode === projectModes.EXPLORATION) {
-      setDatasetSource("benchmark");
+      setDatasetSource("file");
     }
     if (props.mode !== projectModes.EXPLORATION) {
-      setDatasetSource("file");
+      setDatasetSource("benchmark");
     }
```

Comment:
Oef, what's going on here? Bugfix? If so, I would be happy to have this in a standalone PR.

Reply:
No, I changed the default screen from showing `upload benchmark data` to `from file`, which makes more sense to me for the validation mode.

Reply:
Oke, makes sense!